services/QuillLMS/client/app/bundles/Staff/components/evidence/configureSettings/imageSection.tsx

```diff
@@ -13,14 +16,38 @@ export const ImageSection = ({
   imageAttributionGuideLink,
   handleSetImageAttribution
 }) => {
+  const [uploadedMediaLink, setUploadedMediaLink] = React.useState<string>(activityPassages[0].image_link);
+
+  function handleDrop (acceptedFiles) {
+    acceptedFiles.forEach(file => {
+      const data = new FormData()
+      data.append('file', file)
+      fetch(`${process.env.DEFAULT_URL}/cms/images`, {
```

Comment:
It would be nice to use the `requestPost` function here, but I realize that's not the pattern we have in other places with this function.